soc/common/riscv-privileged/Kconfig

```diff
@@ -2,10 +2,19 @@
 # architecture specification
 
 # Copyright (c) 2017 Jean-Paul Etienne <fractalclone@gmail.com>
+# Copyright (c) 2024 sensry.io
 # SPDX-License-Identifier: Apache-2.0
 
 config RISCV_VECTORED_MODE
 	bool "Should the SOC use vectored mode"
 	depends on RISCV_PRIVILEGED
 	help
 	  Should the SOC use vectored mode.
+
+config RISCV_HAS_HART_ID
+	bool "Use of hartID (mhartid/shartid/uarthid)"
+	default y
+	help
+	  Depending on selected kernel run mode, the appropriate hartID
+	  will be used. Should be enabled when booting in machine mode; if disabled
+	  then booting on first core will be assumed.
```

Comment:
Ditto: "first core" doesn't say much, really. You meant to say that core with `hartid = 0` will be used, right?

Reply:
yes, agree.